src/cc2olx/models.py

```diff
@@ -282,6 +286,16 @@ def get_resource_content(self, identifier):
                     )
                 )
                 return "html", {"html": html}
+            elif "web_resources" not in str(res_filename):
+                # Keep track of this file and later copy into static dir
```

Comment:
can we make this comment a little more elaborate?

Reply:
Like what exactly it means if the  web_resource folder is not present.

Reply:
Comment updated with more clear explanation.